src/LeftNav/LeftNav.less

```diff
@@ -99,3 +103,10 @@
     display: none;
   }
 }
+
+
+.expand {
```

Comment:
nit: could this be `.animateWidth`/`.transitionWidth` or something along those lines. on first read above, it seemed like it was applying expanded width styles